### ISSUE TITLE
Issue with new API Permissions not granted

### ISSUE THREAD

**Issue by bogdanurse** [microsoft/navcontainerhelper#1967]
PLEASE DO NOT INCLUDE ANY PASSWORDS OR TOKENS IN YOUR ISSUE!!!

**Describe the issue**
Since 2.0.13 we can’t log in to the environment anymore, without granting consent for the API Permissions added as presented in the image below. Could you add a switch to enable this api permissions just when we need it? Or another fix for it?

**Scripts used to create container and cause the issue**
```
Create-AadAppsForNav
```

**Full output of scripts**
```
The scripts are not generating an error
...
```

**Screenshots**
![image](https://user-images.githubusercontent.com/41284730/121867574-c09ed800-cd08-11eb-897a-9a4c794d1195.png)
![image](https://user-images.githubusercontent.com/41284730/121867643-d7ddc580-cd08-11eb-9d6c-fec58787e83e.png)

**Additional context**
- does it happen all the time?
- Yes
- did it use to work?
- Yes

**Comment by freddydk** [comment]
A fix is underway.
First version will add a parameter for the API permissions
Later I might add a function to actually perform the grant of permissions.

**Comment by bogdanurse** [comment]
Great. Thank you!

**Comment by freddydk** [comment]
latest preview has the fix

**Comment by freddydk** [comment]
Shipped in 2.0.15
